src/client/src/utils/constants.ts

```diff
@@ -78,7 +80,8 @@ const EXTENSION_MODULES = {
   TELEMETRY: "Telemetry",
   VALIDATOR: "Validator",
   VSCODEUI: "VSCodeUI",
-  DEPENDENCYCHECKER: "DependencyChecker"
+  DEPENDENCYCHECKER: "DependencyChecker",
+  CoreTSModule: "CoreTSModule"
```

Comment:
NIT: For consistency, should it be `CORETS` instead of `CoreTsModule`?

Reply:
Good Catch, naming should be capitalized. Will update that in my PR.

Also, I was thinking CoreTS, however, your refactor file has a similar name "coreTemplateStudio.ts", this was why I named it CoreTSModule. What do you think? Should I stick with "CORETSMODULE" or change to "CORETS"?

Reply:
PR updated